static/locales/en-US.yaml

```diff
@@ -48,6 +48,9 @@ Global:
   Counts:
     Video Count: 1 video | {count} videos
     Channel Count: 1 channel | {count} channels
+    Subscriber Count: 1 subscriber | {count} subscribers
+    View Count: 1 view | {count} views
+    Watching Count: 1 watching | {count} watching
```

Comment:
Though not mentioned by
https://kazupon.github.io/vue-i18n/guide/pluralization.html#pluralization

When zero value not provided and count is zero, the plural version is used (I tested locally) (for en at least)